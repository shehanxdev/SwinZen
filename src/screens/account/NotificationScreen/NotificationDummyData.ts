import { NotificationDataType } from '@sz/models';

// TODO:: dummy data until API integration and firebase integration
export const notificationDummyData: NotificationDataType[] = [
  {
    time: new Date('2023-05-05T09:10:30.087Z'),
    message: 'Your video is successfully uploaded now. You can check  the video now...',
    read: false,
  },
  {
    time: new Date('2023-05-05T11:10:30.087Z'),
    message: 'Your video is successfully uploaded now. You can check  the video now...',
    read: true,
  },
  {
    time: new Date('2023-05-04T10:10:30.087Z'),
    message: 'Your video is successfully uploaded now. You can check  the video now...',
    read: false,
  },
  {
    time: new Date('2023-05-04T10:10:30.087Z'),
    message: 'Your video is successfully uploaded now. You can check  the video now...',
    read: false,
  },
  {
    time: new Date('2023-05-04T10:10:30.087Z'),
    message: 'Your video is successfully uploaded now. You can check  the video now...',
    read: true,
  },
  {
    time: new Date('2023-05-01T10:10:30.087Z'),
    message: 'Your video is successfully uploaded now. You can check  the video now...',
    read: false,
  },
  {
    time: new Date('2023-05-01T10:10:30.087Z'),
    message: 'Your video is successfully uploaded now. You can check  the video now...',
    read: true,
  },
  {
    time: new Date('2023-04-15T10:10:30.087Z'),
    message: 'Your video is successfully uploaded now. You can check  the video now...',
    read: false,
  },
];
