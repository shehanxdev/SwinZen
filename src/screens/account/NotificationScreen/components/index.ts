export * from './NotificationCard';
export * from './SectionHeader';
