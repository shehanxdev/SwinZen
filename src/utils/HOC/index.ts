export * from './linear-gradient-wrapper';
