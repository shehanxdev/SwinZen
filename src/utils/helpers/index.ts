export * from './common.helpers';
export * from './notification.helpers';
export * from './user.helpers';
