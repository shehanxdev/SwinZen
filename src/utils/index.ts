export * from './unit-tests';
export * from './validations';
