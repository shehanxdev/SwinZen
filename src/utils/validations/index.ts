export * from './auth';
export * from './info';
export * from './account';
