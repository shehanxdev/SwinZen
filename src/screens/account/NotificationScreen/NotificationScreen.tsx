import React from 'react';
import { Text, View } from 'react-native';

import { Link } from '@sz/components';
import { tw } from '@sz/config';
import { NavigationService } from '@sz/services';

import { BaseScreen } from '../components';

export function NotificationScreen() {
  return (
    <BaseScreen>
      <View style={tw`m-auto`}>
        <Text style={tw`m-10`}>Notification Screen</Text>
        <Link text="Go Back" onPress={() => NavigationService.goBack()} />
      </View>
    </BaseScreen>
  );
}
