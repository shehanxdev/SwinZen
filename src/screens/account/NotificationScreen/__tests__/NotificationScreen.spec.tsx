import * as React from 'react';

import { renderWithProviders } from '@sz/utils';

import { NotificationScreen } from '../NotificationScreen';

describe('Notification Screen Screen', () => {
  const testID = 'NotificationScreenTestID';

  const getRenderedScreen = () => renderWithProviders(<NotificationScreen />);

  it(`should render Notification Screen correctly`, () => {
    const renderer = getRenderedScreen();
    const renderTree = renderer.toJSON();
    expect(renderTree).toMatchSnapshot();
  });

  it('should find the Notification Screen via testID', () => {
    const { getByTestId } = getRenderedScreen();
    const foundScreen = getByTestId(testID);
    expect(foundScreen).toBeTruthy();
  });
});
