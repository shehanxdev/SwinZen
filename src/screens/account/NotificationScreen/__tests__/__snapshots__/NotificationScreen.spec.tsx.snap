// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notification Screen Screen should render Notification Screen correctly 1`] = `
<BVLinearGradient
  colors={
    [
      4279917603,
      4278984207,
      4278650631,
    ]
  }
  endPoint={
    {
      "x": 0.5,
      "y": 1,
    }
  }
  locations={null}
  startPoint={
    {
      "x": 0.5,
      "y": 0,
    }
  }
  style={
    {
      "flexBasis": "0%",
      "flexGrow": 1,
      "flexShrink": 1,
      "position": "relative",
    }
  }
  testID="NotificationScreenTestID"
>
  <Image
    resizeMode="repeat"
    source={
      {
        "testUri": "../../../src/assets/images/grassBackground.jpg",
      }
    }
    style={
      {
        "flexBasis": "0%",
        "flexGrow": 1,
        "flexShrink": 1,
        "opacity": 0.1,
        "position": "absolute",
      }
    }
  />
  <RCTSafeAreaView
    style={
      {
        "height": "100%",
        "paddingTop": 50,
      }
    }
  >
    <View
      onLayout={[Function]}
      style={
        [
          {
            "flexBasis": "0%",
            "flexGrow": 1,
            "flexShrink": 1,
          },
          {
            "paddingBottom": 0,
          },
        ]
      }
    >
      <View
        style={
          {
            "marginLeft": 20,
            "marginRight": 20,
            "marginTop": 12,
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "#F8F9FA",
                "lineHeight": 24,
              },
              {
                "fontFamily": "SourceSansPro-SemiBold",
                "fontSize": 16,
                "fontWeight": "600",
                "textAlign": "auto",
              },
            ]
          }
        >
          You have
          <Text
            style={
              [
                {
                  "color": "#A2FD2F",
                  "lineHeight": 24,
                },
                {
                  "fontFamily": "SourceSansPro-SemiBold",
                  "fontSize": 16,
                  "fontWeight": "600",
                  "textAlign": "center",
                },
              ]
            }
          >
             5 new 
          </Text>
          notifications today!
        </Text>
      </View>
      <RCTScrollView
        data={
          [
            {
              "data": [
                {
                  "message": "Your video is successfully uploaded now. You can check  the video now...",
                  "read": false,
                  "time": 2023-05-05T09:10:30.087Z,
                },
                {
                  "message": "Your video is successfully uploaded now. You can check  the video now...",
                  "read": true,
                  "time": 2023-05-05T11:10:30.087Z,
                },
              ],
              "title": "Last Week",
            },
            {
              "data": [
                {
                  "message": "Your video is successfully uploaded now. You can check  the video now...",
                  "read": false,
                  "time": 2023-05-04T10:10:30.087Z,
                },
                {
                  "message": "Your video is successfully uploaded now. You can check  the video now...",
                  "read": false,
                  "time": 2023-05-04T10:10:30.087Z,
                },
                {
                  "message": "Your video is successfully uploaded now. You can check  the video now...",
                  "read": true,
                  "time": 2023-05-04T10:10:30.087Z,
                },
              ],
              "title": "Last Week",
            },
            {
              "data": [
                {
                  "message": "Your video is successfully uploaded now. You can check  the video now...",
                  "read": false,
                  "time": 2023-05-01T10:10:30.087Z,
                },
                {
                  "message": "Your video is successfully uploaded now. You can check  the video now...",
                  "read": true,
                  "time": 2023-05-01T10:10:30.087Z,
                },
              ],
              "title": "05/01/2023",
            },
            {
              "data": [
                {
                  "message": "Your video is successfully uploaded now. You can check  the video now...",
                  "read": false,
                  "time": 2023-04-15T10:10:30.087Z,
                },
              ],
              "title": "04/15/2023",
            },
          ]
        }
        getItem={[Function]}
        getItemCount={[Function]}
        keyExtractor={[Function]}
        onContentSizeChange={[Function]}
        onLayout={[Function]}
        onMomentumScrollBegin={[Function]}
        onMomentumScrollEnd={[Function]}
        onScroll={[Function]}
        onScrollBeginDrag={[Function]}
        onScrollEndDrag={[Function]}
        renderItem={[Function]}
        scrollEventThrottle={50}
        stickyHeaderIndices={[]}
      >
        <View>
          <View
            onFocusCapture={[Function]}
            onLayout={[Function]}
            style={null}
          >
            <View
              style={
                {
                  "alignItems": "flex-start",
                  "marginBottom": 4,
                  "marginLeft": 20,
                  "marginRight": 20,
                  "marginTop": 32,
                }
              }
            >
              <Text
                style={
                  [
                    {
                      "color": "#F0F7F0",
                      "lineHeight": 28,
                    },
                    {
                      "fontFamily": "SourceSansPro-SemiBold",
                      "fontSize": 21,
                      "fontWeight": "600",
                      "textAlign": "center",
                    },
                  ]
                }
              >
                Last Week
              </Text>
            </View>
          </View>
          <View
            onFocusCapture={[Function]}
            onLayout={[Function]}
            style={null}
          >
            <View>
              <View
                accessibilityState={
                  {
                    "busy": undefined,
                    "checked": undefined,
                    "disabled": undefined,
                    "expanded": undefined,
                    "selected": undefined,
                  }
                }
                accessibilityValue={
                  {
                    "max": undefined,
                    "min": undefined,
                    "now": undefined,
                    "text": undefined,
                  }
                }
                accessible={true}
                collapsable={false}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  {
                    "marginLeft": 16,
                    "marginRight": 16,
                    "opacity": 1,
                  }
                }
              >
                <View
                  style={
                    {
                      "flexDirection": "row",
                      "gap": 12,
                      "marginTop": 16,
                    }
                  }
                >
                  <View
                    style={
                      {
                        "marginTop": 8,
                      }
                    }
                  >
                    <SvgMock
                      fill="#A2FD2F"
                      height={8}
                      width={8}
                    />
                  </View>
                  <View
                    style={
                      {
                        "flexBasis": "0%",
                        "flexGrow": 1,
                        "flexShrink": 1,
                      }
                    }
                  >
                    <Text
                      style={
                        [
                          {
                            "color": "#E9ECEF",
                            "lineHeight": 24,
                          },
                          {
                            "fontFamily": "SourceSansPro-Regular",
                            "fontSize": 16,
                            "fontWeight": "normal",
                            "textAlign": "auto",
                          },
                        ]
                      }
                    >
                      Your video is successfully uploaded now. You can check  the video now...
                    </Text>
                  </View>
                </View>
                <View
                  style={
                    {
                      "flexDirection": "row",
                      "gap": 11,
                      "justifyContent": "flex-end",
                      "marginBottom": 16,
                      "marginTop": 5,
                    }
                  }
                >
                  <View
                    style={
                      {
                        "marginTop": 4,
                      }
                    }
                  >
                    <SvgMock
                      fill="#DAFEAC"
                      height={17}
                      width={17}
                    />
                  </View>
                  <Text
                    style={
                      [
                        {
                          "color": "#F8F9FA",
                          "lineHeight": 24,
                        },
                        {
                          "fontFamily": "SourceSansPro-Regular",
                          "fontSize": 16,
                          "fontWeight": "normal",
                          "textAlign": "auto",
                        },
                      ]
                    }
                  >
                    5 days ago
                  </Text>
                </View>
              </View>
              <View
                style={
                  {
                    "backgroundColor": "#6C757D",
                    "height": 1,
                    "marginLeft": 16,
                    "marginRight": 16,
                  }
                }
              />
            </View>
          </View>
          <View
            onFocusCapture={[Function]}
            onLayout={[Function]}
            style={null}
          >
            <View
              accessibilityState={
                {
                  "busy": undefined,
                  "checked": undefined,
                  "disabled": undefined,
                  "expanded": undefined,
                  "selected": undefined,
                }
              }
              accessibilityValue={
                {
                  "max": undefined,
                  "min": undefined,
                  "now": undefined,
                  "text": undefined,
                }
              }
              accessible={true}
              collapsable={false}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                {
                  "marginLeft": 16,
                  "marginRight": 16,
                  "opacity": 1,
                }
              }
            >
              <View
                style={
                  {
                    "flexDirection": "row",
                    "gap": 12,
                    "marginTop": 16,
                  }
                }
              >
                <View
                  style={
                    {
                      "marginTop": 8,
                    }
                  }
                >
                  <SvgMock
                    fill="#6C757D"
                    height={8}
                    width={8}
                  />
                </View>
                <View
                  style={
                    {
                      "flexBasis": "0%",
                      "flexGrow": 1,
                      "flexShrink": 1,
                    }
                  }
                >
                  <Text
                    style={
                      [
                        {
                          "color": "#6C757D",
                          "lineHeight": 24,
                        },
                        {
                          "fontFamily": "SourceSansPro-Regular",
                          "fontSize": 16,
                          "fontWeight": "normal",
                          "textAlign": "auto",
                        },
                      ]
                    }
                  >
                    Your video is successfully uploaded now. You can check  the video now...
                  </Text>
                </View>
              </View>
              <View
                style={
                  {
                    "flexDirection": "row",
                    "gap": 11,
                    "justifyContent": "flex-end",
                    "marginBottom": 16,
                    "marginTop": 5,
                  }
                }
              >
                <View
                  style={
                    {
                      "marginTop": 4,
                    }
                  }
                >
                  <SvgMock
                    fill="#6C757D"
                    height={17}
                    width={17}
                  />
                </View>
                <Text
                  style={
                    [
                      {
                        "color": "#6C757D",
                        "lineHeight": 24,
                      },
                      {
                        "fontFamily": "SourceSansPro-Regular",
                        "fontSize": 16,
                        "fontWeight": "normal",
                        "textAlign": "auto",
                      },
                    ]
                  }
                >
                  5 days ago
                </Text>
              </View>
            </View>
          </View>
          <View
            onFocusCapture={[Function]}
            onLayout={[Function]}
            style={null}
          />
          <View
            onFocusCapture={[Function]}
            onLayout={[Function]}
            style={null}
          >
            <View
              style={
                {
                  "alignItems": "flex-start",
                  "marginBottom": 4,
                  "marginLeft": 20,
                  "marginRight": 20,
                  "marginTop": 32,
                }
              }
            >
              <Text
                style={
                  [
                    {
                      "color": "#F0F7F0",
                      "lineHeight": 28,
                    },
                    {
                      "fontFamily": "SourceSansPro-SemiBold",
                      "fontSize": 21,
                      "fontWeight": "600",
                      "textAlign": "center",
                    },
                  ]
                }
              >
                Last Week
              </Text>
            </View>
          </View>
          <View
            onFocusCapture={[Function]}
            onLayout={[Function]}
            style={null}
          >
            <View>
              <View
                accessibilityState={
                  {
                    "busy": undefined,
                    "checked": undefined,
                    "disabled": undefined,
                    "expanded": undefined,
                    "selected": undefined,
                  }
                }
                accessibilityValue={
                  {
                    "max": undefined,
                    "min": undefined,
                    "now": undefined,
                    "text": undefined,
                  }
                }
                accessible={true}
                collapsable={false}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  {
                    "marginLeft": 16,
                    "marginRight": 16,
                    "opacity": 1,
                  }
                }
              >
                <View
                  style={
                    {
                      "flexDirection": "row",
                      "gap": 12,
                      "marginTop": 16,
                    }
                  }
                >
                  <View
                    style={
                      {
                        "marginTop": 8,
                      }
                    }
                  >
                    <SvgMock
                      fill="#A2FD2F"
                      height={8}
                      width={8}
                    />
                  </View>
                  <View
                    style={
                      {
                        "flexBasis": "0%",
                        "flexGrow": 1,
                        "flexShrink": 1,
                      }
                    }
                  >
                    <Text
                      style={
                        [
                          {
                            "color": "#E9ECEF",
                            "lineHeight": 24,
                          },
                          {
                            "fontFamily": "SourceSansPro-Regular",
                            "fontSize": 16,
                            "fontWeight": "normal",
                            "textAlign": "auto",
                          },
                        ]
                      }
                    >
                      Your video is successfully uploaded now. You can check  the video now...
                    </Text>
                  </View>
                </View>
                <View
                  style={
                    {
                      "flexDirection": "row",
                      "gap": 11,
                      "justifyContent": "flex-end",
                      "marginBottom": 16,
                      "marginTop": 5,
                    }
                  }
                >
                  <View
                    style={
                      {
                        "marginTop": 4,
                      }
                    }
                  >
                    <SvgMock
                      fill="#DAFEAC"
                      height={17}
                      width={17}
                    />
                  </View>
                  <Text
                    style={
                      [
                        {
                          "color": "#F8F9FA",
                          "lineHeight": 24,
                        },
                        {
                          "fontFamily": "SourceSansPro-Regular",
                          "fontSize": 16,
                          "fontWeight": "normal",
                          "textAlign": "auto",
                        },
                      ]
                    }
                  >
                    6 days ago
                  </Text>
                </View>
              </View>
              <View
                style={
                  {
                    "backgroundColor": "#6C757D",
                    "height": 1,
                    "marginLeft": 16,
                    "marginRight": 16,
                  }
                }
              />
            </View>
          </View>
          <View
            onFocusCapture={[Function]}
            onLayout={[Function]}
            style={null}
          >
            <View>
              <View
                accessibilityState={
                  {
                    "busy": undefined,
                    "checked": undefined,
                    "disabled": undefined,
                    "expanded": undefined,
                    "selected": undefined,
                  }
                }
                accessibilityValue={
                  {
                    "max": undefined,
                    "min": undefined,
                    "now": undefined,
                    "text": undefined,
                  }
                }
                accessible={true}
                collapsable={false}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  {
                    "marginLeft": 16,
                    "marginRight": 16,
                    "opacity": 1,
                  }
                }
              >
                <View
                  style={
                    {
                      "flexDirection": "row",
                      "gap": 12,
                      "marginTop": 16,
                    }
                  }
                >
                  <View
                    style={
                      {
                        "marginTop": 8,
                      }
                    }
                  >
                    <SvgMock
                      fill="#A2FD2F"
                      height={8}
                      width={8}
                    />
                  </View>
                  <View
                    style={
                      {
                        "flexBasis": "0%",
                        "flexGrow": 1,
                        "flexShrink": 1,
                      }
                    }
                  >
                    <Text
                      style={
                        [
                          {
                            "color": "#E9ECEF",
                            "lineHeight": 24,
                          },
                          {
                            "fontFamily": "SourceSansPro-Regular",
                            "fontSize": 16,
                            "fontWeight": "normal",
                            "textAlign": "auto",
                          },
                        ]
                      }
                    >
                      Your video is successfully uploaded now. You can check  the video now...
                    </Text>
                  </View>
                </View>
                <View
                  style={
                    {
                      "flexDirection": "row",
                      "gap": 11,
                      "justifyContent": "flex-end",
                      "marginBottom": 16,
                      "marginTop": 5,
                    }
                  }
                >
                  <View
                    style={
                      {
                        "marginTop": 4,
                      }
                    }
                  >
                    <SvgMock
                      fill="#DAFEAC"
                      height={17}
                      width={17}
                    />
                  </View>
                  <Text
                    style={
                      [
                        {
                          "color": "#F8F9FA",
                          "lineHeight": 24,
                        },
                        {
                          "fontFamily": "SourceSansPro-Regular",
                          "fontSize": 16,
                          "fontWeight": "normal",
                          "textAlign": "auto",
                        },
                      ]
                    }
                  >
                    6 days ago
                  </Text>
                </View>
              </View>
              <View
                style={
                  {
                    "backgroundColor": "#6C757D",
                    "height": 1,
                    "marginLeft": 16,
                    "marginRight": 16,
                  }
                }
              />
            </View>
          </View>
          <View
            onFocusCapture={[Function]}
            onLayout={[Function]}
            style={null}
          >
            <View
              accessibilityState={
                {
                  "busy": undefined,
                  "checked": undefined,
                  "disabled": undefined,
                  "expanded": undefined,
                  "selected": undefined,
                }
              }
              accessibilityValue={
                {
                  "max": undefined,
                  "min": undefined,
                  "now": undefined,
                  "text": undefined,
                }
              }
              accessible={true}
              collapsable={false}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                {
                  "marginLeft": 16,
                  "marginRight": 16,
                  "opacity": 1,
                }
              }
            >
              <View
                style={
                  {
                    "flexDirection": "row",
                    "gap": 12,
                    "marginTop": 16,
                  }
                }
              >
                <View
                  style={
                    {
                      "marginTop": 8,
                    }
                  }
                >
                  <SvgMock
                    fill="#6C757D"
                    height={8}
                    width={8}
                  />
                </View>
                <View
                  style={
                    {
                      "flexBasis": "0%",
                      "flexGrow": 1,
                      "flexShrink": 1,
                    }
                  }
                >
                  <Text
                    style={
                      [
                        {
                          "color": "#6C757D",
                          "lineHeight": 24,
                        },
                        {
                          "fontFamily": "SourceSansPro-Regular",
                          "fontSize": 16,
                          "fontWeight": "normal",
                          "textAlign": "auto",
                        },
                      ]
                    }
                  >
                    Your video is successfully uploaded now. You can check  the video now...
                  </Text>
                </View>
              </View>
              <View
                style={
                  {
                    "flexDirection": "row",
                    "gap": 11,
                    "justifyContent": "flex-end",
                    "marginBottom": 16,
                    "marginTop": 5,
                  }
                }
              >
                <View
                  style={
                    {
                      "marginTop": 4,
                    }
                  }
                >
                  <SvgMock
                    fill="#6C757D"
                    height={17}
                    width={17}
                  />
                </View>
                <Text
                  style={
                    [
                      {
                        "color": "#6C757D",
                        "lineHeight": 24,
                      },
                      {
                        "fontFamily": "SourceSansPro-Regular",
                        "fontSize": 16,
                        "fontWeight": "normal",
                        "textAlign": "auto",
                      },
                    ]
                  }
                >
                  6 days ago
                </Text>
              </View>
            </View>
          </View>
          <View
            onFocusCapture={[Function]}
            onLayout={[Function]}
            style={null}
          />
          <View
            onFocusCapture={[Function]}
            onLayout={[Function]}
            style={null}
          >
            <View
              style={
                {
                  "alignItems": "flex-start",
                  "marginBottom": 4,
                  "marginLeft": 20,
                  "marginRight": 20,
                  "marginTop": 32,
                }
              }
            >
              <Text
                style={
                  [
                    {
                      "color": "#F0F7F0",
                      "lineHeight": 28,
                    },
                    {
                      "fontFamily": "SourceSansPro-SemiBold",
                      "fontSize": 21,
                      "fontWeight": "600",
                      "textAlign": "center",
                    },
                  ]
                }
              >
                05/01/2023
              </Text>
            </View>
          </View>
          <View
            style={
              {
                "height": 0,
              }
            }
          />
        </View>
      </RCTScrollView>
    </View>
  </RCTSafeAreaView>
</BVLinearGradient>
`;
